### PR TITLE
Improve populator error messages

### PR DESCRIPTION
**Pull request by dkubb**
This branch improves `Spree::OrderPopulator` error messages.

### CODE REVIEW

**Comment by dkubb** [comment]
@mbj this is ready for review.

File: core/lib/spree/testing_support/common_rake.rb [hunk 1]
```diff
@@ -16,10 +16,10 @@
     Spree::InstallGenerator.start ["--lib_name=#{ENV['LIB_NAME']}", "--auto-accept", "--migrate=false", "--seed=false", "--sample=false", "--quiet", "--user_class=#{args[:user_class]}"]
 
     puts "Setting up dummy database..."
-    system("bundle exec rake db:drop db:create db:migrate > #{File::NULL}")
+    system("bundle exec rake db:drop db:create db:migrate > #{File::NULL} 2>&1")
```
Comment (mbj) [inline core/lib/spree/testing_support/common_rake.rb]: I'd argue we want all the output. Especially as this command fails silently (that should be fixed separately, ideally in this branch). And if it fails it should be able to print its reason, reducing the debug cycles from having to repro the failure.

File: core/lib/spree/testing_support/common_rake.rb [hunk 2]
```diff
@@ -16,10 +16,10 @@
...
 
     puts "Precompiling assets..."
-    system("bundle exec rake assets:precompile > #{File::NULL}")
+    system("bundle exec rake assets:precompile > #{File::NULL} 2>&1")
```
Comment (mbj) [inline core/lib/spree/testing_support/common_rake.rb]: See my comment in: https://github.com/MountainRoseHerbs/spree/pull/70/files#r31773109

File: core/lib/spree/testing_support/common_rake.rb (hunk 1 above)
Comment (dkubb) [inline core/lib/spree/testing_support/common_rake.rb]: Would it be possible to split initialization from the test run? I find the output from these two commands over overshadows the actual test results and makes things harder to read.

Comment (mbj) [inline core/lib/spree/testing_support/common_rake.rb]: I agree this output is noisy. I think filtering away stdout and keeping stderr while raising on nonzero exit (to be added) is the way to go.

Fixing the test initialization to be separate from the test run is the better (best) option, but that is a more invasive change that needs to be discussed with upstream. Lets just shadow stdout and raise on error as a middle way?

**Comment by mbj** [comment]
@dkubb You intentionally removed the output redirect change on test setup? In that case I'm ready to merge the reminder diff.

**Comment by dkubb** [comment]
> You intentionally removed the output redirect change on test setup

@mbj Yes.

**Pull request merged**